include/core/parameters_multiphysics.h

```diff
@@ -63,6 +63,12 @@ namespace Parameters
     manual
   };
 
+  enum class EpsilonVerbosity
+  {
+    quiet,
+    verbose
+  };
```

Comment:
Doxygen commenting

```suggestion
  /**
   * @brief Verbosity options for the epsilon parameter.
   */
  enum class EpsilonVerbosity
  {
    /// Epsilon related information will not be displayed on terminal
    quiet,
    /// Epsilon value will be displayed on terminal for every steady and
    /// transient iteration.
    verbose
  };
```

Reply:
Thanks for the suggested briefs and comments